### PR TITLE
fix: handle authorization correctly

### LINKED ISSUE #197: Check user rights on Argos
As a user, I have a warning if my token doesn't match with my rights "privateSync" status in "users" table. 

### Motivation

Prevent users to be stuck.

### PR DESCRIPTION
**Pull request by neoziro**
screen shot 2017-03-19 at 14 27 51

Fix #197

### CODE REVIEW

File: src/review/modules/components/AuthorizationNotice.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      <Layout container align="center" justify="center">
+        <Layout item xs={12} sm>
+          Your GitHub authentification is outdated, please authenticate to fix it.
+        </Layout>
+        <Layout item sm={false}>
+          <Button
+            accent
+            compact
+            component={Link}
+            href={user.privateSync ? '/auth/github-private' : '/auth/github-public'}
```
Comment (oliviertassinari) [inline src/review/modules/components/AuthorizationNotice.js]: Should we have a pathname mapping somewhere like we have with rails?

File: src/review/modules/components/AuthorizationNotice.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            raised
+          >
+            {'Authenticate to GitHub'}
```
Comment (oliviertassinari) [inline src/review/modules/components/AuthorizationNotice.js]: Useless complexity 😋

File: src/review/modules/components/AuthorizationNotice.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import React, { PropTypes } from 'react'
+import { connect } from 'react-redux'
+import { createStyleSheet } from 'jss-theme-reactor'
+import withStyles from 'material-ui/styles/withStyles'
```
Comment (oliviertassinari) [inline src/review/modules/components/AuthorizationNotice.js]: I have made an update to merge those two dependencies but it's not released yet.
```jsx
import { createStyleSheet, withStyles } from 'material-ui/styles'
```

File: src/review/modules/components/AuthorizationNotice.js (hunk 1 above)
Comment (neoziro) [inline src/review/modules/components/AuthorizationNotice.js]: Why not :)

**Pull request merged**